ptsa/data/TimeSeriesX.py

```diff
@@ -106,9 +123,6 @@ def to_hdf(self, filename, mode='w'):
             raise RuntimeError("You must install h5py to save as HDF5")
 
         with h5py.File(filename, mode) as hfile:
-            hfile.attrs['ptsa_version'] = ptsa_version
```

Comment:
It looks like you removed the `ptsa_version` and `created` attributes which is causing the TravisCI build to fail. Can you add these back?